lib/sapience/rails/engine.rb

```diff
@@ -73,8 +73,8 @@ class Engine < ::Rails::Engine
         Sapience::Extensions::ActionController::LogSubscriber.attach_to :action_controller
         # Sapience::Extensions::ActiveSupport::MailerLogSubscriber.attach_to :action_mailer
         if defined?(ActiveRecord)
-          Sapience::Extensions::ActiveRecord::LogSubscriber.attach_to :active_record
           Sapience::Extensions::ActiveRecord::Notifications.use
+          ActiveRecord::Base.logger.level = Sapience.config.log_level_active_record
```

Comment:
This won't work like in my case when there is no logger when we reach here. We need a nil check here as to avoid race conditions.